# file_fail.py (gabim i qellimshem)
# Sigurohu qe ekziston 'test.txt' ne te njejtin folder me kete
skript
f = open("data/test.txt", "r")
print(f.read())
f.close()